client/src/app/components/stock-detail-header/stock-detail-header.component.scss

```diff
@@ -4,26 +4,33 @@
     display: flex;
     align-items: center;
     margin-bottom: 20px;
+    justify-content: space-between;
+    align-items: center;
+    
+    div { 
```

Comment:
we should not have this as a div but rather a class name. It is bad practice to style a naked div.

.stock-headline type thing